src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -1218,8 +1215,91 @@ public string GetSummary()
     [BestFriend]
     internal abstract class CalibratorTrainerBase : ICalibratorTrainer
     {
+        public sealed class DataStore : IEnumerable<DataStore.DataItem>
+        {
+            public readonly struct DataItem
```

Comment:
>public readonly struct DataItem [](start = 12, length = 31)

Really vague name for a class visible throughout the assembly.

Reply:
As far as I see it is a nested class inside the internal `CalibratorTrainerBase`?

Reply:
A) it's part of internal class.
B) I'm making it private protected.
C) I'm just moving this code around to hide it, I don't have much intent in improving it's right now

---
In reply to: [258183106](https://github.com/dotnet/machinelearning/pull/2601#discussion_r258183106) [](ancestors = 258183106)